array_shift.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const insertShiftArray = function(arr, v) {
+  let mIndex = Math.ceil(arr.length / 2);
```

Comment:
"Without utilizing any of the built-in methods available to your language"  <- Be careful about this